.github/workflows/tests.yml

```diff
@@ -33,6 +142,34 @@ jobs:
         GIT_URL: ${{ secrets.GIT_URL }}
       run: |
         bin/setup_daq
+    - name: Loading docker images
+      run: |
+        touch .build_built
```

Comment:
too ugly/mess to have inline here, encapsulate in a script

Reply:
Added t oa script